test/include/seqan3/test/pretty_printing.hpp

```diff
@@ -0,0 +1,32 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Contains an hacky overload for the googletest PrintTo function.
+ * \author Svenja Mehringer <svenja.mehringer AT fu-berlin.de>
+ */
```

Comment:
Now we have also doxygen documentation in `test/`? I think it's useful here. Is there already a `make` target to build this?

Reply:
For the stuff in `test/include` we do have docs

Reply:
Oh, good!